apps/design/design.md

```diff
@@ -140,6 +141,10 @@ Since we have representation from the Go SDK team, we might have them go bridge
 the current implementation using some of that code so its possible for our users to store the
 cert in Keyvault.
 
-## Notes:
+## Logging
 
-Do we need: AcquireTokenSilent()??  Seems like we could just bake this into other acquire calls automatically????
\ No newline at end of file
+All logging is done using the `fmt` package. For errors, see [error design](../errors/error_design.md).
```

Comment:
"Logging by `fmt`" implies printing to stdout, which clients don't do.
```suggestion
For errors, see [error design](../errors/error_design.md).
```
Actually, clients communicate by return values instead of logging, so although I like the rest of this content, it's a little awkward. However, it's good information and vacuously true today, and it would be concretely true of any future logging, which wouldn't include PII.